libminifi/test/unit/EnvironmentUtilsTests.cpp

```diff
@@ -117,6 +117,9 @@ TEST_CASE("multithreaded environment manipulation", "[getenv][setenv][unsetenv]"
               utils::Environment::unsetEnvironmentVariable(env_name.c_str());
               break;
             }
+          default: {
+            throw std::runtime_error("Operation value must be modulo 3");
```

Comment:
The idea was throwing an unhandled runtime_error which will terminate by default eventually. What would be the benefit of calling terminate explicitly?

Reply:
If it terminates, then it's impossible to accidentally catch, log and proceed on a programming error. It would be best if it prevented compilation, but since this is a runtime condition, we can only detect the condition at runtime. Crashing the program can not be ignored, which should prevent releasing with a programming error.